site/en/docs/web-platform/navigation-api/index.md

```diff
@@ -92,90 +108,252 @@ Links might come and go on your page, and they're not the only way users can nav
 
-## Transition
+## Deciding how to handle a navigation
 
-When your code calls `transitionWhile()` from within its "navigate" listener, it informs the browser that it's now preparing the page for the new, updated state; and that the navigation may take some time. The `Promise` you pass to `transitionWhile()` tells the browser how long the navigation takes.
+The `navigateEvent` contains a lot of information about the navigation that you can use to decide whether to intercept a particular navigation.
 
-As such, this API introduces a semantic concept that the browser understands: an SPA navigation is currently occurring, over time, changing the document from a previous URL and state to a new one.
-This has a number of potential benefits, including accessibility: browsers can surface the beginning, end, or potential failure of a navigation.
-Chrome, for example, activates its native loading indicator, and allows the user to interact with the stop button. (This doesn't currently happen when the user navigates via the back/forward buttons, but that [will be fixed soon][loading-crbug].)
+The key properties are:
 
-### Transition Success and Failure
+`canIntercept`
+: If this is false, you can't intercept the navigation.
+Cross-origin navigations, and cross-document traversals cannot be intercepted.
 
-After the "navigate" event completes, the URL being navigated to will take effect.
-This happens immediately, even if you've called `transitionWhile()`.
+`destination.url`
+: Probably the most important piece of information to consider when handling the navigation.
 
-{% Aside 'caution' %}
-This means `navigation.currentEntry`, `location.href`, etc. will update immediately. This impacts things like relative URL resolution when fetching new data or loading new subresources.
+`hashChange`
+: True if the navigation is same-document, and the hash is the only part of the URL that's different to the current URL.
+In modern SPAs, the hash should be for linking to different parts of the current document. So, if `hashChange` is true, you probably don't need to intercept this navigation.
 
-Many web and native applications immediately update the page with some sort of placeholder for the incoming content. But if you don't also immediately update the page's content, it will be out of sync with your application's programmatic view of the current entry and URL, which can be tricky.
+`downloadRequest`
+: If this is true, the navigation was initiated by a link with a `download` attribute.
+In most cases, you don't need to intercept this.
 
-These issues are being [discussed on GitHub](https://github.com/WICG/navigation-api/issues/66).
-{% endAside %}
+`formData`
+: If this isn't null, then this navigation is part of a POST form submission.
+Make sure you take this into account when handling the navigation.
+If you only want to handle GET navigations, avoid intercepting navigations where `formData` is not null.
```

Comment:
Maybe mention that actually intercepting this is pretty common, to progressively enhance `<form>` submissions into Ajax? Forward-link to the `#form-submissions` heading?

Reply:
Shall do. Although, I wonder how many folks will do this vs handle the form as they do today, then perform a replacement navigation when the POST is complete.